Install Flyve MDM and FusionInventory plugins

ingluife opened the pull request:
Hi, @flyve-mdm/backend-development.

In this changes i configured  the cloning process of Flyvemdm and Fusioninventory. This process is executed after the GLPI's instalation process and starting automatically.
For that i add the environment variables necesary for the cloning. Also i configure a container additional for Mosquitto service required by Flyvemdm.

For Mosquitto service i will have configure and compile the auth-plugin.

The mosquitto's container is based in the Docker official image in:
https://hub.docker.com/_/eclipse-mosquitto/ 

### References
Ref #21
Closes #23

ingluife committed:
build(docker): add flyvemdm and fusioninventory

add variables for flyvemdm cloning
add variables for fusioninventory cloning

Signed-off-by: Luis F. Gonzalez <lgonzalez@teclib.com>

ingluife committed:
build(docker): configure mosquitto container

add mosquitto directory
add Dockerfile Mosquitto
add variables for mosquitto's ports
add configuration minimal for mosquitto

Signed-off-by: Luis F. Gonzalez <lgonzalez@teclib.com>

DIOHz0r commented on .env:
```diff
@@ -14,6 +14,7 @@ APACHE_TAG=2.4
 CLONNING_METHOD=ssh
 GLPI_BRANCH=9.2/bugfixes
 GLPI_SOURCE=git@github.com:glpi-project/glpi.git
+GLPI_PLUGINS_DIRECTORY=/plugins
```
There is no need to set a plugin directory var because it'll be always the plugin directory inside GLPI home path, please remove/adjust that on the rest of the code

ingluife replied:
Ok, i will change it

DIOHz0r commented on docker/php/cliinstall.sh:
```diff
@@ -9,14 +9,31 @@ if [ "$(ls -A $HOMEPATH)" ]; then
+    git clone --progress --depth=1 $FLYVEMDM_SOURCE -b $FLYVEMDM_BRANCH $HOMEPATH$GLPI_PLUGINS_DIRECTORY$FLYVEMDM_PATH
+    cd $HOMEPATH$GLPI_PLUGINS_DIRECTORY$FLYVEMDM_PATH
+    echo "Installing packages...";
+    su - www-data -c 'composer install --no-dev'
```
we must install develop vendors for Flyvemdm

ingluife replied:
That's right.

DIOHz0r commented on docker/php/cliinstall.sh:
```diff
@@ -9,14 +9,31 @@ if [ "$(ls -A $HOMEPATH)" ]; then
+if [ -d $HOMEPATH$GLPI_PLUGINS_DIRECTORY$FLYVEMDM_PATH ] && [ "$(ls -A $HOMEPATH$GLPI_PLUGINS_DIRECTORY$FLYVEMDM_PATH)" ]; then
+    echo "The directory's Flyvemdm exist"
+    echo "If you want reinstall, remove the directory $FLYVEMDM_PATH and $FUSIONINVENTORY_PATH, and try again."
+    cd $HOMEPATH$GLPI_PLUGINS_DIRECTORY$FLYVEMDM_PATH
```
I think this is not necessary if you remove the else statement and simplify the if block to only print the message.

ingluife replied:
Ok, i will do it.

ingluife committed:
build(docker): edit cliinstall GLPI

add ssh directory variable
remove plugins directory variable
edit installing packages in cloning process

Signed-off-by: Luis F. Gonzalez <lgonzalez@teclib.com>

ingluife commented:
Hi, @flyve-mdm/backend-development.

This is ready for Test in LINUX and MAC.
